back-end/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.6
+
+WORKDIR /usr/src/app
+
+COPY requirements.txt ./
+RUN pip install --upgrade pip
```

Comment:
This is like, nitpick level 11 out of a possible 10…but maybe put an explicit pip version in here, like `pip install --upgrade pip==16.0.0` or whatever the latest is. Pip and setuptools have suffered occasional instability the last couple years so knowing exactly what you're getting can help avoid future bugs.

Reply:
Would be great to get rid of the Dockerfile completely as per our convo @npentrel